packages/@aws-cdk/aws-stepfunctions/lib/states/map.ts

```diff
@@ -205,11 +249,28 @@ export class Map extends State implements INextable {
   }
 
   /**
-   * Render Parameters in ASL JSON format
+   * Render ItemSelector in ASL JSON format
    */
-  private renderParameters(): any {
+  private renderItemSelector(): any {
+    return FieldUtils.renderObject({
+      ItemSelector: this.itemSelector,
+    });
+  }
+
+  /**
+   * Render ItemProcessor in ASL JSON format
+   */
+  private renderItemProcessor(): any {
+
+    var iterator = this.renderIterator();
+
     return FieldUtils.renderObject({
-      Parameters: this.parameters,
```

Comment:
We should still be rendering this, even though they're deprecated; that, or we need to make `parameters` reroute to `ItemProcessor`.